pkg/policies/opa/rego/gcp/google_container_cluster/AC-GC-IS-CC-M-0367.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "gkeControlPlanePublicAccess",
+    "file": "gkeControlPlanePublicAccess.rego",
+    "template_args": null,
+    "severity": "Medium",
+    "description": "GKE Control Plane is exposed to few public IP addresses using master-authorized-network-config",
+    "reference_id": "AC-GC-IS-CC-M-0367",
```

Comment:
Can we use the new rule ref id format here? :)

Reply:
@harkirat22

Reply:
@gaurav-gogia . I would prefer to make the rule-reference id update only once its formally documented as our previous updates did break some of the end user's pipelines.

Reply:
@harkirat22 I see, sounds good